clang/lib/CodeGen/SwiftCallingConv.cpp

```diff
@@ -814,7 +816,10 @@ static ABIArgInfo classifyType(CodeGenModule &CGM, CanQualType type,
     auto &layout = CGM.getContext().getASTRecordLayout(record);
 
     if (mustPassRecordIndirectly(CGM, record))
-      return ABIArgInfo::getIndirect(layout.getAlignment(), /*byval*/ false);
+      return ABIArgInfo::getIndirect(
+          layout.getAlignment(),
+          /*AddrSpace*/ CGM.getContext().getTargetAddressSpace(LangAS::Default),
+          /*byval*/ false);
```

Comment:
```suggestion
          /*AddrSpace=*/ CGM.getContext().getTargetAddressSpace(LangAS::Default),
          /*byval=*/ false);
```

as clang-format prefers

Reply:
Reworked altogether.